gemrb/unhardcoded/bg2ee/damage.2da

```diff
@@ -1,16 +1,16 @@
 2DA V1.0
 *
 	MAIN		SPARKS		GRADIENT
```

Comment:
missing column header?

Reply:
looks like a few of these files have that issue

Reply:
still missing for HoW

Reply:
Thanks, missed beautifing some of the research bits.